django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -67,6 +68,7 @@ def escape_curly_brackets(text: str):
 class ChatConsumer(AsyncWebsocketConsumer):
     full_reply: ClassVar = []
     citations: ClassVar = []
+    external_citations: ClassVar = []
```

Comment:
External citations as in not the documents from user? So gov.uk would be considered external citations?